pokedex-with-bit/pods/doc.yml

```diff
@@ -1,6 +1,23 @@
-!DataURIPbIndexer
+!BinaryPbIndexer
 with:
-  index_filename: doc.gzip
+  index_filename: doc.gz
 metas:
-  name: doc_indexer  # a customized name
-  workspace: $WORKDIR
\ No newline at end of file
+  name: doc_idx
+  workspace: $WORKDIR
+requests:
+  on:
+    ControlRequest:
+      - !ControlReqDriver {}
+    SearchRequest:
+      - !KVSearchDriver
+        with:
+          granularity_range: [0, 0]
+          adjacency_range: [0, 1]
```

Comment:
```suggestion
          - !ExcludeQL
            with:
              fields:
                - embedding
```

Reply:
is it removing or adding on top of it? I think we need to keep the `adjacency_range`